### PR TITLE
Service lb

### PR DESCRIPTION
**Pull request by puja108**
Did not include full description of deploying a full second IC, but mentioned use cases and what needs to be kept in mind.

### CODE REVIEW

File: content/guides/ervices-of-type-loadbalancer-and-multiple-ingress-controllers/index.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+#### SSL Termination
+
+There's three annotations you can set to configure SSL termination.
+
+```YAML
+service.beta.kubernetes.io/aws-load-balancer-ssl-cert: arn:aws:acm:us-east-1:123456789012:certificate/12345678-1234-1234-1234-123456789012
+```
+
+The first one depicts the ARN of the certificate you want to use. You can either upload the certificateto IAM or create it within AWS Certificate Manager.
```
Comment (jgsqware) [inline content/guides/ervices-of-type-loadbalancer-and-multiple-ingress-controllers/index.md]: typo ` certificateto` to ` certificate to`

File: content/guides/ervices-of-type-loadbalancer-and-multiple-ingress-controllers/index.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+or 
+
+```YAML
+kubernetes.io/ingress.class: "nginx-internal"
+```
+
+Not specifying the annotation will lead to multiple ingress controllers claiming the same ingress. Specifying a value which does not match the class of any existing ingress controllers will result in all ingress controllers ignoring the ingress.
+
+Further note that if you are running additional Ingress Controllers that are based you might need to configure them so their Ingress Class does not collide with the class of our default NGINX Ingress Controller. For the community supported NGINX Ingress Controller this is described in the [official documentation]((https://github.com/kubernetes/ingress-nginx#running-multiple-ingress-controllers)).
```
Comment (jgsqware) [inline content/guides/ervices-of-type-loadbalancer-and-multiple-ingress-controllers/index.md]: `that are based` Based on what?

Comment (puja108) [reply content/guides/ervices-of-type-loadbalancer-and-multiple-ingress-controllers/index.md]: removed

File: content/guides/ervices-of-type-loadbalancer-and-multiple-ingress-controllers/index.md [hunk 3]
```diff
@@ -19,6 +19,8 @@ __Note__ that this functionality cannot be used on premises.
 
 Setting the `type` field of your service to `LoadBalancer` will result in your Service being exposed by a dynamically provisioned load balancer.
 
+You can do this with any Service within you cluster, including Services that expose several ports.
```
Comment (jgsqware) [inline content/guides/ervices-of-type-loadbalancer-and-multiple-ingress-controllers/index.md]: typo `you` `your`

File: content/guides/ervices-of-type-loadbalancer-and-multiple-ingress-controllers/index.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+    - hostname: a54cae28bd42b11e7b2c7020a3f15370-27798109.eu-central-1.elb.amazonaws.com
+```
+
+### Customizing the External Load Balancer
+
+As we are currently only supporting the AWS cloud, this section will focus on the custom options you can set on the AWS Elastic Load Balancer via a Service of type `LoadBalancer`. You can configure these options by adding annotations to the service.
+
+#### Internal ELB
+
+If you want the ELB to be available only within your VPC (can be extended to other VPC by VPC peering) use following annotation:
```
Comment (fgimenez) [inline content/guides/ervices-of-type-loadbalancer-and-multiple-ingress-controllers/index.md]: `use the following`

File: content/guides/ervices-of-type-loadbalancer-and-multiple-ingress-controllers/index.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+metadata:
+  name: my-service
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-internal: 0.0.0.0/0
+[...]
+```
+
+#### SSL Termination
+
+There's three annotations you can set to configure SSL termination.
```
Comment (fgimenez) [inline content/guides/ervices-of-type-loadbalancer-and-multiple-ingress-controllers/index.md]: `There are three`

File: content/guides/ervices-of-type-loadbalancer-and-multiple-ingress-controllers/index.md [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+service.beta.kubernetes.io/aws-load-balancer-ssl-cert: arn:aws:acm:us-east-1:123456789012:certificate/12345678-1234-1234-1234-123456789012
+```
+
+The first one depicts the ARN of the certificate you want to use. You can either upload the certificate to IAM or create it within AWS Certificate Manager.
+
+```YAML
+service.beta.kubernetes.io/aws-load-balancer-backend-protocol: (https|http|ssl|tcp)
+```
+
+The second annotation specifies which protocol a pod speaks. For HTTPS and SSL, the ELB will expect the pod to authenticate itself over the encrypted connection.
```
Comment (fgimenez) [inline content/guides/ervices-of-type-loadbalancer-and-multiple-ingress-controllers/index.md]: `speaks to`

File: content/guides/ervices-of-type-loadbalancer-and-multiple-ingress-controllers/index.md [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+```YAML
+service.beta.kubernetes.io/aws-load-balancer-backend-protocol: http
+service.beta.kubernetes.io/aws-load-balancer-ssl-ports: "443,8443"
+```
+
+In the above example, if the service contained three ports, `80`, `443`, and `8443`, then `443` and `8443` would use the SSL certificate, but `80` would just be proxied HTTP.
+
+## Using Multiple Ingress Controllers {#multiple-ingress}
+
+By default a cluster in Giant Swarm is bootstrapped with a default Ingress Controller based on NGINX. This Ingress Controller is registered with the default `nginx` Ingress Class.
```
Comment (fgimenez) [inline content/guides/ervices-of-type-loadbalancer-and-multiple-ingress-controllers/index.md]: I wonder how this worked https://github.com/giantswarm/giantswarm/wiki/Using-multiple-ingress-controllers, the new IC had the same annotation as the default one...

Comment (puja108) [reply content/guides/ervices-of-type-loadbalancer-and-multiple-ingress-controllers/index.md]: most probably both ICs were acting and exposing the service

**Pull request merged**